src/download.rs

```diff
@@ -60,7 +52,7 @@ fn get_file_name(response: &Response, orig_url: &reqwest::Url) -> String {
         .or_else(|| from_url(orig_url))
         .unwrap_or_else(|| "index".to_string());
 
-    let filename = filename.split(std::path::is_separator).next_back().unwrap();
+    let filename = sanitize_filename::sanitize(&filename);
```

Comment:
I see `sanitize-filename` has a `replacement` option. Maybe we can tell it to replace by `"_"` since that's what most browsers seem to do?